.github/workflows/ci.yaml

```diff
@@ -636,3 +640,116 @@ jobs:
         asset_path: cartographer-conventions-samples-bundle.tar
         asset_name: cartographer-conventions-samples-bundle-${{ steps.get_version.outputs.VERSION }}.tar
         asset_content_type: application/x-tar
+    - uses: vmware-tanzu/carvel-setup-action@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Install crane
+      run: |
+        cd $(mktemp -d -t kind.XXXX)
+        curl -L https://github.com/google/go-containerregistry/releases/download/v0.8.0/go-containerregistry_Linux_x86_64.tar.gz | tar -xz
+        chmod +x ./crane
+        sudo mv ./crane /usr/local/bin
+        cd -
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - name: Relocate bundle to public registry
+      run: |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+
+        version=${{ steps.get_version.outputs.VERSION }}
+
+        imgpkg copy --tar cartographer-conventions-bundle.tar --to-repo "ghcr.io/${{ github.repository }}/bundle"
+        crane tag "ghcr.io/${{ github.repository }}/bundle" "${version}"
+        digest=$(crane digest "ghcr.io/${{ github.repository }}/bundle:${version}")
+
+        scratch=$(mktemp -d -t bundle.XXXX)
+        mkdir -p ${scratch}
+        imgpkg pull -b "ghcr.io/${{ github.repository }}/bundle:${version}@${digest}" -o ${scratch}
+
+        cp hack/boilerplate.ytt.txt cartographer-conventions.yaml
+        kbld -f ${scratch}/config/cartographer-conventions.yaml -f ${scratch}/.imgpkg/images.yml \
+          >> cartographer-conventions.yaml
+
+        cp hack/boilerplate.ytt.txt cartographer-conventions-package.yaml
+        ytt -f dist/package.yaml -f dist/package.values.yaml \
+          --data-value version=${version} \
+          --data-value image="ghcr.io/${{ github.repository }}/bundle:${version}@${digest}" \
+          >> cartographer-conventions-package.yaml
```

Comment:
soo here's we're producing the final `package.yaml`, but I noticed there's also a `package-install.yaml` with a values file there too - should we also generate and upload in a later step that too?

Reply:
The PackageInstall resource needs to be managed by the user. We have one in the repo that we use for CI, but I wouldn't expect a user to use that one.